python/mead/export.py

```diff
@@ -18,6 +18,7 @@ def main():
     parser.add_argument('--model_version', help='model_version', default=1)
     parser.add_argument('--output_dir', help='output dir', default='./models')
     parser.add_argument('--beam', help='beam_width', default=30, type=int)
+    parser.add_argument("--preproc", help="use tensorflow preprocessing", default=False, action="store_true", dest="preproc")
```

Comment:
It would be more general, and more inline with Baseline's extensibility paradigm to use a more general approach to defining the exporter (like `export_type`)